### DISCUSSION TITLE
Push updates

### DISCUSSION THREAD

**Discussion by shaurya415** [community/community#63911] [Repositories]
### Select Topic Area

Question

### Body

How to push updates to private repository from local machine in windows

**Comment by ghost** [comment]
Greetings,

I am here to help you today,
First of all you need to download this software (easier way) : [Git Bash](https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.3/Git-2.41.0.3-64-bit.exe)
Then login using your github account in which your repository exists.

Then in terminal:
1. cd directory path
2. git add -A (Add updated files)
3. git commit -m "some message" (commit files)
4. git push

**All Done!**

> Have a nice day ahead

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
